.azure-pipelines/build-template.yml

```diff
@@ -97,20 +101,48 @@ jobs:
   - checkout: self
     submodules: true
   - script: |
+      set -ex
       ./autogen.sh
-      fakeroot dpkg-buildpackage -b -us -uc -Tbinary-syncd-vs -j$(nproc) && cp ../*.deb .
-    displayName: "Compile sonic sairedis"
+      fakeroot debian/rules DEB_CONFIGURE_EXTRA_FLAGS='--enable-code-coverage' CFLAGS="" CXXFLAGS="" binary-syncd-vs && cp ../*.deb .
+    displayName: "Compile sonic sairedis with coverage enabled"
   - script: |
       sudo cp azsyslog.conf /etc/rsyslog.conf
       sudo service rsyslog restart
     displayName: "Update rsyslog.conf"
   - ${{ if eq(parameters.run_unit_test, true) }}:
     - script: |
+        set -ex
+        git clone https://github.com/Spacetown/gcovr.git
+        cd gcovr/
+        git checkout origin/recursive_search_file
```

Comment:
@kcudnik , looks like the branch recursive_search_file is removed, can we change to the other branch?

Reply:
please update template